api/swagger-spec/v1.json

```diff
@@ -5971,7 +5971,7 @@
         "allowMultiple": false
        },
        {
-        "type": "",
+        "type": "string",
```

Comment:
Oh wow, that feels like bringing up something to life from the dead. That part of the spec wasn't well defined in 1.2 which is why it was revised in 2.0. I honestly don't recall if `multi` as it is in 2.0 existed in 1.2, but I believe it did not (was one of the feature requests).